src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -27,6 +27,14 @@
       String template,
       String outputPath,
       TargetLanguage language) {
+    // Note that internally, this is overridable using a "discovery" section in the service config,
+    // which is not available externally. See:
+    //   https://github.com/googleapis/toolkit/issues/933
+    String discoveryApiName = model.getServiceConfig().getName();
```

Comment:
This is actually service name. And I would suggest naming the first segment as apiName without mentioning discovery.